index.html

```diff
@@ -21,26 +21,102 @@ <h2 class="notoc">Contents</h2>
+  review.</p>
+  </li>
+  <li>When a document is both reasonably stable and still flexible enough to
+  allow changes where issues are identified.</li>
+  <li>When new features are added after a document has already received wide
+  review (perhaps requesting a review limited to the new features).</li>
+  </ul>
+  </section>
 
 <section id="who_to_ask_for_review">
-<h2>Who to ask for review?</h2>
- 
-<p>Much of this document focuses on how and when to conduct horizontal reviews, but they are only a subset of a full wide review, which must also include other stakeholders including Web developers, technology providers and implementers not active in the Working Group, external groups and standards organizations working on related areas, etc. Here is a list of suggestions:</p>
+<h2>Who to ask for wide review?</h2>
+
+ <p>The objective is to ensure that the entire set of stakeholders of the Web
+ community, including the general public, have  adequate notice of the progress
+ of the Working Group and are able to actually perform reviews of and provide
+ comments on the specification. When considering <a
+ href="https://www.w3.org/Guide/transitions">requests to advance the maturity
+ level of the document</a>, the Director will consider who has been explicitly
+ offered a reasonable opportunity to review the document.</p>
+
+<p>You must publish an updated technical report, with the Status of the Document
+indicating clearly that you're looking for <em>wide review</em>, before inviting
+review from other stakeholders. Our <a
+href='https://github.com/w3c/transition-notifier/blob/main/notify.js'>tooling</a>
+monitors publications and propagates notifications to <a
+href="mailto:public-review-announce@w3.org">public-review-announce@w3.org</a>
+automatically (for example, <a
+href='https://lists.w3.org/Archives/Public/public-review-announce/2021Jun/0008.html'>Candidate
+Recommendation Snapshot: Payment Request API (Call for Wide Review)</a>).</p>
+
+<p>You should then reach out to:</p>
+
 <ul>
-<li> Groups listed in the WG's charter, especially those who manage dependencies.</li>
-<li> Groups jointly responsible for a particular document (if any)  (duh!).</li>
-<li> All horizontal groups (listed below).  If it appears that one of those is not relevant (and is not listed in your charter), talk to them informally to confirm that.</li>
-<li> Other groups, at your discretion, even if not in the WG charter, including other W3C groups, external organizations and SSOs, implementers, application developers, etc.</li>
-<li>The general public. Consider using blog posts, social media, or other ways of alerting the public to your document and requesting feedback.</li>
+<li>the groups listed in the WG's charter, especially those who manage
+dependencies.</li>
+<li>the groups jointly responsible for a particular document (if any).</li>
+<li>the <a href="#how_to_get_horizontal_review">horizontal groups</a>.  If it
```

Comment:
```suggestion
<li>the <a href="https://www.w3.org/Guide/process/charter.html#horizontal-review">horizontal groups</a>
using the <a href="#how_to_get_horizontal_review">method described below</a>.  If it
```
<a href="https://www.w3.org/Guide/process/charter.html#horizontal-review">horizontal groups</a>